pkg/certificates/mock_certificates/Oser.go

```diff
@@ -1,69 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Im not too sure about combining Oser and Rsaer and into a common mock. They have no connection to each other 🤔